hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/contracts/execution/MirrorEvmTxProcessorFacadeImpl.java

```diff
@@ -53,6 +55,7 @@ public class MirrorEvmTxProcessorFacadeImpl implements MirrorEvmTxProcessorFacad
     public MirrorEvmTxProcessorFacadeImpl(
             final MirrorEntityAccess entityAccess,
             final MirrorNodeEvmProperties evmProperties,
+            final TracingProperties tracingProperties,
```

Comment:
`MirrorOperationTracer` is a bean, so inject it directly and don't leak its dependencies. It's probably the only `HederaEvmOperationTracer` bean so you can also inject the higher level interface.

```suggestion
            final MirrorOperationTracer operationTracer,
```